app/controllers/application_controller/tags.rb

```diff
@@ -111,7 +111,8 @@ def tagging_edit_tags_reset
     @title = _('Tag Assignment')
     if tagging_explorer_controller?
       @refresh_partial = "layouts/tagging"
-      replace_right_cell(:action => @sb[:action]) if params[:button]
+      action = x_active_tree == :svcs_tree && params[:button] == "reset" ? "tag" : @sb[:action]
```

Comment:
So the problem here was that `@sb[:action]` in this particular case was not set properly? Or was not set at all? Do I understand the problem correctly?

If so, shouldn't we rather fix `@sb[:action]` so that it's set correctly? The above fix would just introduce special behavior for one specific tree into a code, which otherwise is written in a generic fashion (more or less).

Reply:
@sb[:action] was not set at all, in this particular case. But maybe since I opened this PR, "things" may have changed. I will check it now again and update this PR. Thank you for good questions! :)

Just for explanation, as I remember, fixing @sb[:action] was ...hell for me. And it looks like you understand it much more than I. So I would ask: another hints/ideas how to fix the bug in a better way? 👼

Reply:
I'm not saying I necessarily have better understanding of the code. I'd just like to avoid creating special if (or condition if you will) for one specific invocation of the tagging code.

What I'd do is look at tagging of different type of objects, where the reset works correctly and see what's different and where the `@sb[:action]` is being set in there.